styles/alert/alert.css

```diff
@@ -17,35 +23,34 @@
 }
 
 .alert-primary {
-  color: rgb(11, 230, 230);
+  color: var(--blue);
   background-color: var(--default);
 }
 .alert-secondary {
-  color: rgb(165, 103, 11);
-  background-color: rgb(245, 245, 185);
+  color: var(--dark-yellow);
+  background-color: var(--light-yellow);
 }
 .alert-success {
-  color: rgb(138, 230, 126);
-  background-color: rgb(63, 146, 63);
+  color: var(--light-green);
+  background-color: var(--dark-green);
 }
 .alert-danger {
-  color: rgb(243, 212, 212);
-  background-color: rgb(245, 115, 115);
+  color: var(--light-red);
+  background-color: var(--dark-red);
 }
-.alert-primary-outline{
-    outline: 3px solid rgb(11, 230, 230);
-    color: rgb(11, 230, 230);
+.alert-primary-outline {
+  outline: 3px solid var(--blue);
+  color: var(--blue);
 }
-.alert-secondary-outline{
-outline: 3px solid  rgb(165, 103, 11);
-    color: rgb(165, 103, 11);
+.alert-secondary-outline {
+  outline: 3px solid var(--dark-yellow);
+  color: var(--dark-yellow);
 }
-.alert-success-outline{
-    outline: 3px solid rgb(63, 146, 63);
-    color: rgb(63, 146, 63);
+.alert-success-outline {
+  outline: 3px solid var(--dark-green);
+  color: var(--dark-green);
 }
```

Comment:
space after each {} would make the code cleaner.

Reply:
thanks , I will do that